test/trento/discovery/policies/cluster_policy_test.exs

```diff
@@ -1184,7 +1184,7 @@ defmodule Trento.Discovery.Policies.ClusterPolicyTest do
                |> ClusterPolicy.handle(nil)
     end
 
-    test "should set the health to critical when the SAPInstance resourece is running the same node" do
+    test "should set the health to critical when the SAPInstance resource is running the same node" do
```

Comment:
I guess @CDimonaco found the same issue a second time here @janvhs  :+1: 

https://github.com/trento-project/web/pull/2650/files

Reply:
:eyes: great ideas are in the air